thermo/EquationsOfState.ipynb

```diff
@@ -0,0 +1,414 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Specific heat at constant pressure\n",
+    "plot(T, p, cpIdeal, cpRK, cpCoolProp, \"Cp [J/kg/K]\")\n",
+    "# Specific heat at constant volume\n",
+    "plot(T, p, cvIdeal, cvRK, cvCoolProp, \"Cv [J/kg/K]\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the case of Ideal gas EoS, the specific heats at constant pressure (C$_{\\rm p}$) and constant volume (C$_{\\rm v}$) are independent of the Temperature. Hence, C$_{\\rm p}$ and C$_{\\rm v}$ for ideal EoS do not change as the temperature is kept constant at T = 300 K in this study.\n",
+    "\n",
+    "Specific heats for R-K EoS follow the trend closely with the Helmholtz EoS to the critical regime. Although the specific heat at constant pressure (C$_{\\rm p}$) shows reasonable agreement with the Helmholtz EoS in sub-critical and supercritical regimes, it inaccurately predicts a very high value near the critical point. The heat capacity at constant pressure (C$_{\\rm p}$) at the critical point is finite. The sudden rise in C$_{\\rm p}$ value in the case of R-K EoS is just a numerical artifact and not a real singularity.\n",
+    "\n",
+    "The specific heat at constant volume (C$_{\\rm v}$), on the other hand, predicts smaller values in the subcritical and critical regime. However, it shows completely wrong values in the super-critical region, making it invalid at very high pressures. It is a known fact that the cubic equations typically fail to predict accurate constant-volume heat capacity in the transcritical region [1]. Certain cubic EoS models have been extended to resolve this discrepancy using crossover models (More information can be found elsewhere [1],[2]).\n",
+    "\n",
+    ">1. R. Span, “Multiparameter Equations of State - An Accurate Source of Thermodynamic Property Data,” Springer Berlin Heidelberg (2000), http://dx.doi.org/10.1007/978-3-662-04092-8\n",
```

Comment:
Can you please avoid using "fancy" quotes here and in other references?

Reply:
Cleaned up.